scripts/gen_dictionary_from_file.py

```diff
@@ -11,40 +11,41 @@
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
 
-    parser.add_argument('-features', action='store',
+    parser.add_argument('-file', action='store',
```

Comment:
```suggestion
    parser.add_argument('-i', action='store',
```
or `-inputfile` maybe.

Reply:
Done - I switched to `-i` everywhere in the script, and the Makefile.